pkg/api/ca.go

```diff
@@ -53,7 +53,15 @@ func CA() certauth.CertificateAuthority {
 		var err error
 		switch viper.GetString("ca") {
 		case "googleca":
-			ca, err = googlecabeta.NewCertAuthorityService()
+			version := viper.GetString("gcp_private_ca_version")
+			switch version {
+			case "v1":
```

Comment:
In the original pr, these where switches and no one caught it...

Reply:
ooops!